projects/storefrontapp-e2e-cypress/cypress/e2e/b2b/regression/quote/quote.e2e-flaky.cy.ts

```diff
@@ -199,17 +199,58 @@ context('Quote', () => {
     });
 
     it('should submit a quote and not be able to add any further items to the quote in checkout', () => {
-      quote.prepareSellerQuote(SALESREP_EMAIL, SALESREP_PASSWORD, buyer);
-      quote.checkQuoteState(quote.STATUS_REQUESTED);
-      quote.submitQuote(quote.STATUS_SALES_REPORTER_SUBMIT);
-      asm.agentSignOut();
-      quote.login(BUYER_EMAIL, BUYER_PASSWORD, BUYER_USER);
-      quote.goToQuoteOverviewPage();
+      quote.prepareQuoteForCheckout(
+        BUYER_EMAIL,
+        BUYER_PASSWORD,
+        BUYER_USER,
+        SALESREP_EMAIL,
+        SALESREP_PASSWORD
+      );
       quote.submitQuote(quote.STATUS_BUYER_CHECKOUT);
       quote.addProductAndCheckForGlobalMessage(
         TEST_PRODUCT_HAMMER_DRILLING_NAME,
         'Not possible to do changes to cart entries. Proceed to checkout'
       );
     });
   });
+
+  describe('Save active cart - create a new cart after submitting a quote (CXSPA-4141)', () => {
+    beforeEach(() => {
+      quote.prepareQuote(
+        TEST_PRODUCT_HAMMER_DRILLING_ID,
+        PRODUCT_AMOUNT_30,
+        true
+      );
+    });
+
+    it('should become a saved cart after editing the submitted quote', () => {
+      quote.clearSavedCarts();
+      quote.prepareSavedCartTemplate(TEST_PRODUCT_HAMMER_DRILLING_ID);
```

Comment:
this is called in each test, please move it to the beforeEach callback

Reply:
for the quote.clearSavedCarts(); you are right! this can be moved to the beforeEach callback. The quote.prepareSavedCartTemplate(TEST_PRODUCT_HAMMER_DRILLING_ID); needs to stay in each test since its called in different situations.